cnxpublishing/tests/testing.ini

```diff
@@ -25,6 +25,8 @@ openstax_accounts.login_path = /login
 openstax_accounts.callback_path = /callback
 openstax_accounts.logout_path = /logout
 
+channel_processing.channels = post_publication, faux_channel
```

Comment:
This did not get added to `development.ini` in the commit. Want an `--amend` of the commit (and a force-push on master) or a tiny-little-PR?